azure-pipelines.yml

```diff
@@ -250,7 +250,7 @@ resources:
     - repository: templates
       type: github
       name: hmcts/azure-devops-templates
-      ref: refs/heads/master
+      ref: refs/heads/optional-dotnet-test-ci-run
```

Comment:
Should this be master?

Reply:
yes for now, we will revert it when cvp sandbox is up